static/coffee/services.coffee

```diff
@@ -157,11 +155,51 @@ services.factory('MessageService', ['$rootScope', '$http', '$httpParamSerializer
       return $http.post('/message/action/' + action + '/', params)
 ])
 
-
 #=====================================================================
-# Incoming message service
+# FAQ service
 #=====================================================================
+services.factory('FaqService', ['$rootScope', '$http', '$httpParamSerializer', ($rootScope, $http, $httpParamSerializer) ->
+  new class FaqService
+
+    #----------------------------------------------------------------------------
+    # Fetch FAQs to use in replies
+    #----------------------------------------------------------------------------
+    fetchFaqs: (search) ->
+      params = @_searchFaqsToParams(search)
+      return $http.get('/faq/search/?' + $httpParamSerializer(params)).then((response) ->
```

Comment:
This was addressed in https://github.com/rapidpro/casepro/pull/117/commits/0cb3bf4cae45b6c1cb3a8d5bacf8e118501823b5